packages/smooth_app/lib/main.dart

```diff
@@ -227,6 +227,8 @@ class SmoothAppGetLanguage extends StatelessWidget {
     final LocalDatabase _localDatabase = context.read<LocalDatabase>();
     AnalyticsHelper.trackStart(_localDatabase, context);
 
+    context.read<UserManagementProvider>().mountCredentials();
```

Comment:
Not `await`? Btw as it's in the init part and we do that only once, do we have to notify the listeners in `mountCredentials`? I don't think so, as it's dangerous to notify while building a `Widget`.

Reply:
no await intended, since the credentials are not needed directly after start, this can run when there is time for that. Because of that that little more efford with the provider

Reply:
Not convinced. All we do in `mountCredentials` is get the credentials from the dao and put them in static. Quick and easy.
OK, technically you are allowed not to wait until it finishes and use the provider for the refresh, but:
* hey, it's init time, let's init once and for all
* the notifylistener will trigger a display refresh just because you didn't want to spend 20 milliseconds `await`ing
* not everything we do is strictly needed at init time - we also set the query country, which is useless as we haven't queried anything yet
* btw `mountCredentials` could be called much earlier, in `_init1` or `_init2`

Reply:
Just applied your suggestions, my first approach was to completely remove the provider, but looking into it again we should keep it to properly reflect logins from for example the login card.